server/session/handler_player_action.go

```diff
@@ -51,6 +51,8 @@ func handlePlayerAction(action int32, face int32, pos protocol.BlockPos, entityR
 			return nil
 		}
 		s.c.ContinueBreaking(cube.Face(face))
+	case protocol.PlayerActionStartItemUseOn, protocol.PlayerActionStopItemUseOn:
+		// TODO: Properly utilize these actions.
```

Comment:
Do we have to?

Reply:
It would be ideal in my opinion. @Sandertv thoughts?